src/main/java/Duke.java

```diff
@@ -6,5 +9,81 @@ public static void main(String[] args) {
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println("\t____________________________________________________________\n"
+                + "\tHello! I'm Duke\n" +
+                "\tWhat can I do for you?\n"
+                + "\t____________________________________________________________\n");
+
+
+        Task[] taskList = new Task[100];
+        int taskCount = 0;
+
+        Scanner in = new Scanner(System.in);
+        String line = in.nextLine();
+
+        while (!line.equals("bye")){
+            String[] words = line.split(" ",2); // split the first word from the rest of the sentence using space as the delimiter
+            if (line.equals("list")) {
+                printListMessage(taskList, taskCount);
+            } else if (words[0].equals("done")) {
+                int taskNum = Integer.parseInt(line.substring(5,6));
+                Task t=taskList[taskNum-1];
+                t.markAsDone();
+                printDoneMessage(t);
+            } else {
+                String taskType = words[0];
+                Task t = new Task(line);
+                switch (taskType){
+                    case "todo":
```

Comment:
Good job for using switch, will recommend to follow the coding standard because the word "case" must by directly under the switch :)